utility/projects/projectLib.js

```diff
@@ -42,23 +68,37 @@ const updateSubmoduleBranch = (outFolder) => {
     const git = require('simple-git')();
     const gitProjectMs2 = require('simple-git')(`${outFolder}/MapStore2`);
 
+    const stableBranch = "2022.02.xx";
```

Comment:
Should we fetch this dynamically, instead? Or perhaps a note in documentation ?

Reply:
We may setup some logic that does this automatically, but I think, for we can keep it like this.
Maybe we can open a dedicated issues for that.

Let me think about the documentation

Reply:
done, please review again